iroha_client/benches/torii.rs

```diff
@@ -12,20 +12,22 @@ use iroha_client::{
     client::{asset, Client},
```

Comment:
I think this should not be in benches `torii.rs` as it is a separate test case to check whole iroha and its TPS for 4 peers.